projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -335,6 +335,12 @@ export interface FeatureTogglesInterface {
    * New "Back To Results" button is added
    */
   a11yFacetsDialogFocusHandling?: boolean;
+
+  /**
+   * Enables the use of the new default URL for saveCart and cloneSavedCart.
+   * This toggle is used in the following files: `OccCartAdapter`, `OccSavedCartAdapter`, `SavedCartOccModule`, `CartBaseOccModule`.
+   */
+  newOccDefaultEndpointForSaveCartAndCloneSavedCart?: boolean;
```

Comment:
It's very nice the JSDocs mention the affected classes. It might be very helpful for customers to know which their possibly customized classes need fixing after the toggle is enabled.

Moreover, I'd love if the toggle name (and description) could be a bit more informative about what are the actual changes in the behavior. (instead of saying ~"the behavior is new" - what does it mean "new"?)

What about `occCartNameAndDescriptionInHttpRequestBody`? Does it express well the nature of the change in the behavior?
```suggestion

  /**
   * In OCC cart requests, it puts parameters of a cart name and cart description
   * into a request body, instead of query params.
   * This toggle is used in the following clases: `OccCartAdapter`, `OccSavedCartAdapter`, `SavedCartOccModule`, `CartBaseOccModule`.
   */
  occCartNameAndDescriptionInHttpRequestBody?: boolean;
```

Reply:
Your proposition is much better, thanks